packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -102,7 +102,7 @@ export const GetDataTrio = ({ value, onAction }: GetDataTrioProps) => {
             onAction?.('print');
           }}
           data-testid="databutton-print"
-          className="w-12 h-12 relative inline-block text-grey-500 rounded active:text-blue-500 active:outline-black-dotted focus:outline-black-dotted hover:bg-grey-50"
+          className="w-12 h-12 relative inline-block text-grey-500 rounded active:text-blue-500 active:outline-dotted active:outline-black focus:outline-dotted focus:outline-black hover:bg-grey-50"
```

Comment:
ignorant Q: Why "bg-grey-50" here but changed to "bg-black/75" below?

Reply:
The opacity class changed: https://tailwindcss.com/docs/upgrade-guide#new-opacity-modifier-syntax

Reply:
And pre-apologies for casually 👀 a draft PR, _but&hellip;_
I did run through most of the items in the upgrade guide and didn't see anything else in my `git grep` testing, apart from the 1-2 comments below. ❤️

Reply:
Heheh, this is basically ready except for me poking around the styles to make sure things visually look as expected so thanks for the early review. What's strange is the opacity classes don't seem to be working as described (you can see this in the FxA settings design guide on the header border) and I think it _might_ be because we have custom colors set? The old syntax still works apparently, wasn't a "breaking" change according to the doc I sent. Still looking into it.